CallsheetLocalizations/pl.xcloc/Localized Contents/pl.xliff

```diff
@@ -1029,7 +1029,8 @@
         <note/>
       </trans-unit>
       <trans-unit id="Preview" xml:space="preserve">
-        <source>Preview</source><target/>
+        <source>Preview</source>
+        <target state="translated">Przegląd<target/>
```

Comment:
@spitfire It's the preview of the changes to the spoiler settings. This could work as Przegląd, no?

Reply:
Not sure about it, I believe we actually use a word "trailer" for that most of the time. I've got to think about it.

Reply:
It's not a movie preview: It's the preview of how the user interface looks for changes to the spoiler settings:

<img width="250" alt="Bildschirmfoto 2024-05-23 um 14 08 26" src="https://github.com/spitfire/callsheet-localizations/assets/18497564/3357269d-3ea9-4923-86b6-83dd0a2f0b66">

Reply:
Przegląd zmian.

Reply:
Oh, then it should simply be "Podgląd"
"Przegląd" is more of an overview.

Reply:
Ah, I'll fix :)

Reply:
Done!